app/assets/stylesheets/govuk_publishing_components/components/_devolved_nations.scss

```diff
@@ -0,0 +1,11 @@
+.gem-c-devolved_nations {
+  background: govuk-colour("light-grey");
+  color: govuk-colour("black");
```

Comment:
You don't need to specify a colour for this element - colours are already set by classes applied to the child elements (links, headings).